external/model-preparation-algorithm/tests/ote_cli/test_ins_seg_cls_il.py

```diff
@@ -0,0 +1,174 @@
+"""Tests for MPA Class-Incremental Learning for instance segmentation with OTE CLI"""
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import os
+
+import pytest
+
+from ote_sdk.test_suite.e2e_test_system import e2e_pytest_component
+
+from ote_cli.registry import Registry
+
```

Comment:
Could you revise this line breaks?

Reply:
That's a good observation. The same test code on the seg side seems to need to be revised later.